models/user.js

```diff
@@ -1,32 +1,95 @@
 const db = require('../db.js');
 
-const publicFieldsFromUsersTable = "`id`, `name`, `level`, `exp`, `elu`, `description` from `users`";
+const publicFieldsFromUsersTable = "id, name, level, exp, elu, genre_id, race_id, class_id, home_id, description, gold, bank_gold, pos_map, pos_x, pos_y, body_id, killed_npcs, killed_users, counter_connected, is_connected, min_hp, max_hp, min_man, max_man, min_sta, max_sta, min_ham, max_ham, min_sed, max_sed, min_hit, max_hit, body_id, head_id, weapon_id, helmet_id, shield_id, pertenece_consejo_real, pertenece_consejo_caos, pertenece_caos, pertenece_real, ciudadanos_matados, criminales_matados";
 
 exports.getAllUsers = function(req, res) {
-    db.get().query('SELECT * FROM user', function (err, results, fields) {
+    db.get().query(`SELECT ${publicFieldsFromUsersTable} FROM user`, function (err, results, fields) {
         if (err) throw err;
         res.status(200).json(results);
     })
 };
 
 exports.getAllByUser = function(req, res) {
     console.log(req)
-    db.get().query('SELECT * FROM user WHERE id = ?', res.userId, function (err, rows) {
+    db.get().query(`SELECT ${publicFieldsFromUsersTable} FROM user WHERE id = ?;`, res.userId, function (err, rows) {
         if (err) throw err;
         res.status(200).json(results);
     })
 };
 
 exports.getAllUsersConnectedCount = function(req, res) {
```

Comment:
In a future we could a list of the users connected, just the name

Reply:
Yes that's another good approach, that one is for the webpage for the section where will appear 
`USERS ONLINE = 1000 `